Packs/MicrosoftGraphIdentityandAccess/ReleaseNotes/1_2_19.md

```diff
@@ -0,0 +1,6 @@
+#### Integrations
+##### Azure Active Directory Identity And Access
+- Added the following parameters for **Certificate Authentication** support for self-deployed mode.
```

Comment:
lets make this focus on the feature. Added support for certificate authentication for self deployed mode, using the following parameters: